### PR TITLE
Jt 72

### PR DESCRIPTION
**Pull request by aalexeen**

### CODE REVIEW

**Review by saffchen — APPROVED** [review]

File: buildSrc/build.gradle.kts [hunk 1]
```diff
@@ -17,3 +17,8 @@ tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
         jvmTarget = "17"
     }
 }
+
+/*
```
Comment (pelmegov) [inline buildSrc/build.gradle.kts:21]: Minor: remove unused code.

File: stock/src/test/java/saffchen/controller/store/StoreControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    void getAll() throws Exception {
+        repository.save(store1);
+        repository.save(store2);
+        perform(MockMvcRequestBuilders.get(REST_URL))
+                .andExpect(status().isOk())
+                .andExpect(content().contentTypeCompatibleWith(MediaType.APPLICATION_JSON))
+                .andExpect(STORE_MATCHER.contentJson(store1, store2));
+    }
+
+    @Test
```
Comment (pelmegov) [inline stock/src/test/java/saffchen/controller/store/StoreControllerTest.java:31]: Minor: empty tests should be removed or implemented.

File: stock/src/test/resources/test-data.sql [hunk 3]
```diff
@@ -0,0 +1,30 @@
+INSERT INTO store(name, description)
+VALUES ('North Shore', 'test description'),
+       ('Burnie', 'test description');
+
+/*SELECT setval('satellite_id_seq', 30, true);*/
```
Comment (pelmegov) [inline stock/src/test/resources/test-data.sql:5]: Minor: commented code.

File: stock/src/test/java/saffchen/controller/store/StoreTestData.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        store1.setProducts(List.of(store1_1, store1_2, store1_3));
+        store2.setProducts(List.of(store2_1, store2_2, store2_3));
+        store3.setProducts(List.of(store3_1, store3_2, store3_3));
+    }
+
+    public static StoreEntity getNew() {
+        return new StoreEntity(null, "New Store", "not defined");
+    }
+
+    public static StoreEntity getUpdated() {
```
Comment (pelmegov) [inline stock/src/test/java/saffchen/controller/store/StoreTestData.java:52]: It doesn't update anything, it just creates a new one with the constant id.

**Review by pelmegov — APPROVED** [review]

File: buildSrc/build.gradle.kts (hunk 1 above)
Comment (aalexeen) [reply buildSrc/build.gradle.kts:21]: done

File: stock/src/test/java/saffchen/controller/store/StoreControllerTest.java (hunk 2 above)
Comment (aalexeen) [reply stock/src/test/java/saffchen/controller/store/StoreControllerTest.java:31]: done

File: stock/src/test/resources/test-data.sql (hunk 3 above)
Comment (aalexeen) [reply stock/src/test/resources/test-data.sql:5]: done

File: stock/src/test/java/saffchen/controller/store/StoreTestData.java (hunk 4 above)
Comment (aalexeen) [reply stock/src/test/java/saffchen/controller/store/StoreTestData.java:52]: the original store is 
`new StoreEntity(STORE_ID1, "North Shore", "test description")`

the updated Store is going to be
`new StoreEntity(STORE_ID1, "North Shore updated", "test description updated")`

**Pull request merged**